docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -0,0 +1,627 @@
+
+Make sure that you have added a `service_config` to the JSON configuration of your webcam with type `data_manager`, as well as a new service with type `data_manager`.
+
+{{< /alert >}}
+
+### Capture data
+
+Your webcam should now be configured to automatically capture images when you are connected to your robot on [the Viam app](https://app.viam.com).
+
+You set the rate of capture in your webcam's service configuration attibute `capture_frequency_hz`.
+If you set this to `.333`, your webcam should export 1 image roughly every 3 seconds.
+
+At this point, grab the toys or any objects you want the robot to be able to differentiate between.
+
+The inventor's kids like playing with puzzle pieces, which come in different shape and color combinations.
+They decided to filter between these puzzle pieces by tagging by shape, but you can filter your objects as you choose.
+
+Hold up the toys to the camera while photos are being taken.
+Try to capture images from different angles and backgrounds.
+Try to get at least 50 images that fit your criteria for each tag.
+
+Go to the [**DATA tab**](https://app.viam.com/data/view?view=images) in the Viam app to see the images captured by your webcam.
+
+Now, use these pictures to train your machine learning model.
+
+### Tag data
+
+Head over to the [**DATA** page](https://app.viam.com/data/view) and select an image captured from your robot.
+After selecting the image, you will see all of the data that is associated with that image.
+
+Add tags for each of the puzzle pieces.
+Type in your desired tag in the Tags section and save the tag.
+Since we wanted to classify our toys by shape, we used “octagon”, “circle”, “triangle”, “oval”, “rectangle”, “pentagon”, “diamond”, and “square”.
+
+Scroll between your images.
+Add tags for each image that shows an object of the corresponding shape.
+You can select tags from the **Recently used** drop down menu.
+Try to have at least 50 images labeled for each tag.
+This is important for the next step.
+
+#### Filter based on tags
+
+Upon completion of tagging your data set, you can now [filter images](/manage/data/view/) according to those tags.
+Head over to the **Filtering** menu and select a tag from the drop down list to view all labeled images.
+
+### Train a model
+
+After tagging and filtering your images, begin training your model.
+
+Click the **Train Model** button.
+Name your model.
+To match the name we used in our [bedtime songs program](#program-your-bedtime-songs-bot), name your model `"shape-classifier-model"`.
+Select **Multi label** as the model type, which accounts for multiple tags.
+
+Then select the tags that you used to label your toys and click **Train Model**.
+
+Read through our guide to [training a new model](/manage/ml/train-model/) for more information.
+
+## Use your ML Model to sing songs to your kids
+
+Now, deploy and configure your ML Model on your robot, and configure your webcam to act as a shape classifier.
```

Comment:
This is a bit draw the owl
![image](https://github.com/viamrobotics/docs/assets/5212232/c7cbb414-446d-4e4a-8f6c-b16f9364f459)

Reply:
done